README.md

```diff
@@ -1,65 +1,256 @@
 
 ```sh
 kubectl minio init
+
+```
+
+## 2) Create a New Tenant
+
+The following `kubectl minio` command creates a MinIO Tenant with 4 nodes, 16
+volumes, and a total capacity  of 16Ti. This configuration requires
+*at least* 16 
+[Persistent Volumes](https://github.com/minio/operator#Local-Persistent-Volumes).
+
+```sh
+   kubectl minio tenant create \
+      --name minio-tenant1
+      --servers 4
+      --volumes 16
+      --capacity 16Ti
+      --namespace minio-tenant1
 ```
 
-Once the MinIO Operator is created, proceed with Tenant creation.
+- The `--name` field specifies the name of the MinIO Tenant and as the prefix for resources
+  deployed as part of the Tenant.
 
-## Tenant Setup
+- The `--servers` field indicates the number of `minio` pods to deploy into the cluster.
+  The cluster *must* have at least one available Node per `minio` pod. 
 
-A Tenant is a MinIO cluster created and managed by the Operator. Before creating tenant, please ensure you have requisite nodes and drives in place and relevant PVs are created.
+- The `--volumes` field indicates the total number of volumes in the Tenant. MinIO
+  generates a Persistent Volume Claim (`PVC`) for each volume and evenly distributes
+  volumes across each `minio` pod. The example above results in 4 volumes per `minio` pod.
 
-In below example, we ask MinIO Operator to create a Tenant yaml with 4 nodes, 16 volumes, and 16 Ti total raw capacity (4 volumes of 1 Ti per node). This means you need to have 4 PVs of 1Ti each per node, with a total of 4 nodes, before attempting to create the MinIO tenant.
+- The `--capacity` field indicates the total capacity of the cluster. MinIO determines the
+  amount of storage to request for each `pvc` by dividing the specified capacity by the
+  total number of volumes in the server. The example above results in 1Ti requested
+  capacity per volume.
 
-Since MinIO has built-in erasure-code support for high-availability and data protection, we recommend the following CSI drivers for high-performance and scalability:
+- The `--namespace` field indicates the namespace onto which MinIO deploys the Tenant. 
+  If omitted, MinIO uses the `Default` namespace.
 
-- [Local PV](https://kubernetes.io/docs/concepts/storage/volumes/#local)
-- [OpenEBS Local PV](https://docs.openebs.io/docs/next/localpv.html)
-- [vSAN Direct](https://blogs.vmware.com/virtualblocks/2020/09/15/whats-new-in-vsan-7-update-1/)
+## 3) Connect to the Tenant
+
+MinIO outputs credentials for connecting to the MinIO Tenant as part of the creation
+process:
 
 ```sh
-kubectl minio tenant create --name tenant1 --servers 4 --volumes 16 --capacity 16Ti
+MinIO Tenant 'minio-tenant1' created
+
+Username: admin 
+Password: abcd1234-abcd-1234-abcd-1234abcd1234
+
+Web interface access: 
+	$ kubectl port-forward svc/minio-tenant1-console 9090:9443
+	Point browser to https://localhost:9090
+
+Object storage access: 
+	$ kubectl port-forward svc/minio-tenant1-hl 9000:9000
```

Comment:
```suggestion
	$ kubectl port-forward svc/minio 9000:9000
```
TLDR; xd - Here service name should be only `minio`


When operator deploys a new tenant, 2 services are created, "minio" clusterIP service (has an IP address assigned) and "minio-tenant1-hl" headless service (doesnt have an IP address assigned), ie:

```
kubectl get svc
NAME                                           TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)             AGE
kubernetes                                     ClusterIP   10.96.0.1       <none>        443/TCP             19d
minio                                          ClusterIP   10.98.115.236   <none>        443/TCP             24m
minio-tenant1-console   ClusterIP   10.97.71.11     <none>        9090/TCP,9443/TCP   23m
minio-tenant1-hl        ClusterIP   None            <none>        9000/TCP            24m
```
It's better for applications to access directly the `minio` service (K8S will LB connections to all minio pods) instead of the headless service, otherwise they will need to implement LB logic in the client side https://medium.com/faun/kubernetes-headless-service-vs-clusterip-and-traffic-distribution-904b058f0dfd

Reply:
@nitisht @Alevsk  this might require a change to the operator PR

This was based on the output from #369 [here](https://github.com/minio/operator/pull/369/files#diff-22fba376bed03b38062f292edd3f54d976d78364ab8a1f3688f5b830acdd0a51R174), so maybe `minioSVCName` needs to be modified?

Reply:
This should now be addressed.